osgbuild/main.py

```diff
@@ -374,41 +375,41 @@ def parse_cmdline_args(argv):
         "-w", "--working-directory",
         help="The base directory to use for temporary files made by the "
         "script. If it is 'TEMP', a randomly-named directory under /tmp "
-        "is used.")
+        "is used. Default: package directory")
 
     prebuild_group = OptionGroup(parser,
                                  "prebuild task options")
     prebuild_group.add_option(
         "--full-extract", action="store_true",
-        help="Fully extract all source files.")
+        help="Fully extract all source files")
     parser.add_option_group(prebuild_group)
 
     rpmbuild_mock_group = OptionGroup(parser, 
                                       "rpmbuild and mock task options")
     rpmbuild_mock_group.add_option(
         "--distro-tag",
         help="The distribution tag to append to the end of the release. "
-        "(Default: osg.el5, osg.el6 or osg.el7, depending on --redhat-release)")
+        "Default: osg.el5, osg.el6, or osg.el7 for EL 5/6/7 respectively")
```

Comment:
”The default tag is ‘osg.elV’, where ‘V’ is the major version of Enterprise Linux for the build.“ (or something like that)